pom.xml

```diff
@@ -228,7 +228,7 @@
     ./python/pyspark/sql/pandas/utils.py, and ./python/setup.py too.
     -->
     <arrow.version>13.0.0</arrow.version>
-    <ammonite.version>2.5.9</ammonite.version>
+    <ammonite.version>2.5.9-2-7633e640</ammonite.version>
```

Comment:
cc @dongjoon-hyun This is the `ammonite` version that supports Scala 2.13.12, it seems the tests have passed, we can wait for an official release of the `ammonite` version.

Reply:
Thank you, @LuciferYang ! Please create a JIRA issue and link to this PR.

Reply:
created SPARK-45330 and SPARK-45331, I will update this PR after ammonite 2.6.0 is released.